packages/react-components/src/button/src/Button.jsx

```diff
@@ -125,11 +125,11 @@ export function InnerButton(props) {
         },
         icon: {
             size: condensed ? size : embeddedIconSize(size),
-            className: "o-ui-button-left-icon"
+            className: "o-ui-button-icon o-ui-button-left-icon"
         },
         text: {
             size: condensed ? condensedTextSize(size) : size,
-            className: "o-ui-button-text",
+            className: "o-ui-button-text o-ui-text-color-inherit",
```

Comment:
I am not sure why we would need this here?

I don't have a problem with text-color-inherit but I am not sure it should be applied de-factor. I see it more for special cases like an Alert